<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpPage3</class>
 <widget class="QWidget" name="HelpPage3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 50);</string>
  </property>
  <widget class="QPushButton" name="BTNHome">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>620</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/Home.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN2">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN9">
   <property name="geometry">
    <rect>
     <x>826</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN5">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN7">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN3">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN4">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN8">
   <property name="geometry">
    <rect>
     <x>724</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>931</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
border:2px solid;
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/workshoponsupply.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>车间电源连接</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/workshoponnosupply.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>车间电源断开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/hscbfault.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>高速断路器故障</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/hscbon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>高速断路器闭合</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/hscboff.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>高速断路器断开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>931</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
border:2px solid;
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>乘客紧急报警单元故障</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/pisfault.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>360</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>紧急报警</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>350</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/pispassengeractive.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>司机当前紧急通话的乘客紧急报警的单元号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/pisdriveractive.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/pisnoactive.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>乘客紧急通信单元正常，未报警</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>520</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>530</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>乘客紧急通信单元离线</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
